packages/paste-website/src/components/tokens-list/TokensListFilter.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Label} from '@twilio-paste/label';
+import {Input} from '@twilio-paste/input';
+import {FilterIcon} from '@twilio-paste/icons/esm/FilterIcon';
+import {Grid, Column} from '@twilio-paste/grid';
+import {Select, Option} from '@twilio-paste/select';
+import {useUID} from '@twilio-paste/uid-library';
+import type {TokensListFilterProps} from './types';
+
+export const TokensListFilter: React.FC<TokensListFilterProps> = ({
+  handleInput,
+  handleFormatChange,
+  handleThemeChange,
+  selectedFormat,
+  selectedTheme,
+}) => {
+  const inputId = useUID();
+  const themeControlId = useUID();
+  const formatControlId = useUID();
+
+  return (
+    <>
+      <Box marginBottom="space80">
+        <Grid gutter="space40">
+          <Column span={6}>
+            <Label htmlFor={inputId} id="test-label">
+              Filter tokens
+            </Label>
+            <Input
+              type="text"
+              id={inputId}
+              aria-labelledby="test-label"
```

Comment:
Suggestion: you don't need the id on the label and aria-labelledby on the input. 

It is enough to have the id on the input and htmlFor on the label.

Reply:
The search input has another ticket to be worked on before the feature branch is merged in, it's not functional right now (it's just there from being copied over from the TokensList component).